services/CurrentTime/include/ble-service-current-time/CurrentTimeService.h

```diff
@@ -43,6 +44,13 @@
  */
 class CurrentTimeService {
 public:
+    mbed::Timer timer;
+
+    const uint8_t MANUAL_TIME_UPDATE             = 1 << 0;
```

Comment:
please mark it static so it doesn't take space into an instance and stays in flash.